httpclient/src/main/java/org/apache/http/client/utils/URIBuilder.java

```diff
@@ -492,8 +492,8 @@ public String toString() {
 
     private static String normalizePath(final String path) {
         String s = path;
-        if (s == null) {
-            return "/";
+        if (s == null || "".equals(s)) {
+            return "";
```

Comment:
I just didn't want to import new dependences such as apache.commons.lang3.StringUtils, when I didn't notice TextUtils. I will use TextUtils, thanks for recommending.